hadoop-hdds/config/src/main/java/org/apache/hadoop/hdds/conf/TimeDurationUtil.java

```diff
@@ -55,7 +56,7 @@ public static long getTimeDurationHelper(String name, String vStr,
     }
 
     long raw = Long.parseLong(vStr);
-    long converted = unit.convert(raw, vUnit.unit());
+    long converted = unit.convert(raw, Objects.requireNonNull(vUnit).unit());
```

Comment:
if vUnit can be null, it would be better to check its nullity right after line 52, and throw an exception there (which can carry a more meaningful error message).

Reply:
@jojochuang this is addressed in latest patch.  Can you please take another look?